modules/admission-plug-ins-about.adoc

```diff
@@ -6,7 +6,7 @@
 [id="admission-plug-ins-about_{context}"]
 = About admission plugins
 
-Admission plugins are used to help regulate how {product-title} {product-version} functions. Admission plugins intercept requests to the master API to validate resource requests and ensure policies are adhered to, after the request is authenticated and authorized. For example, they are commonly used to enforce security policy, resource limitations or configuration requirements.
+Admission plugins intercept requests to the master API to validate resource requests. After a request is authenticated and authorized, the admission plugins ensure that any associated policies are adhered to. For example, they are commonly used to enforce security policy, resource limitations or configuration requirements.
```

Comment:
Just a suggestion, but do you think that s/followed/adhered to changes the meaning too much?